gtars/src/uniwig/mod.rs

```diff
@@ -430,10 +552,98 @@ pub fn uniwig_main(
                 }
             }
         }
+        }
+    );
+
+    let vec_strings = vec!["start", "core", "end"];
+
+    match output_type {
+        "wig" => {
+            println!("Combining Wig Files");
+
+            for location in vec_strings.iter(){
+
+                write_combined_wig_files(*location, output_type, bwfileheader, &chromosomes);
+
+            }
+
+
+
+        }
+        _ => {
+
+        }
+
     }
+
+
+    println!("FINISHED");
+
     Ok(())
 }
 
+fn fixed_core_wiggle_bam(p0: &Vec<i32>, p1: &Vec<i32>, p2: i32, p3: i32) -> (Vec<u32>, Vec<i32>) {
+    println!("smooth_fixed_start_end_wiggle_bam");
+
+
+    let mut v_coordinate_positions: Vec<i32> = Vec::new(); // these are the final coordinates after any adjustments
+    let mut v_coord_counts: Vec<u32> = Vec::new(); // u8 stores 0:255 This may be insufficient. u16 max is 65535
+
+    return (v_coord_counts, v_coordinate_positions);
+}
+
+fn smooth_fixed_start_end_wiggle_bam(p0: &Vec<i32>, p1: i32, p2: i32, p3: i32) -> (Vec<u32>, Vec<i32>) {
+    println!("smooth_fixed_start_end_wiggle_bam");
+
+
+    let mut v_coordinate_positions: Vec<i32> = Vec::new(); // these are the final coordinates after any adjustments
+    let mut v_coord_counts: Vec<u32> = Vec::new(); // u8 stores 0:255 This may be insufficient. u16 max is 65535
+
+    return (v_coord_counts, v_coordinate_positions);
```

Comment:
Needless return.. its more rust and idiomatic to just return the tuple

```rust
(v_coord_counts, v_coordinate_positions)
```

Reply:
done